src/components/shared/SelectRecipients/SelectRecipientsDialog.tsx

```diff
@@ -55,7 +57,22 @@ export function SelectRecipientsDialogUI(props: SelectRecipientsDialogUIProps) {
     const { t } = useI18N()
     const classes = useStylesExtends(useStyles(), props)
     const { items, disabledItems } = props
-
+    const [search, setSearch] = React.useState('')
+    const [, inputRef] = useCapturedInput((newText) => {
```

Comment:
`useCapturedInput` to prevent event capturing. E.g. prevent input 'i' to trigger the twitter private message box stretch